include/shared.h

```diff
@@ -330,7 +330,7 @@ ssize_t ft_post_tx(struct fid_ep *ep, fi_addr_t fi_addr, size_t size,
 		struct fi_context* ctx);
 ssize_t ft_rx(struct fid_ep *ep, size_t size);
 ssize_t ft_tx(struct fid_ep *ep, fi_addr_t fi_addr, size_t size, struct fi_context *ctx);
-ssize_t ft_inject(struct fid_ep *ep, size_t size);
+ssize_t ft_inject(struct fid_ep *ep, fi_addr_t fi_addr, size_t size);
```

Comment:
I'm not disagreeing with this change, but can you provide the justification for this change in the commit message?  It's not obvious why it was made.